ci-operator/step-registry/aws-load-balancer/pre-install-rosa/aws-load-balancer-pre-install-rosa-commands.sh

```diff
@@ -51,7 +56,14 @@ CR_DIR="/tmp/albo-credrequests"
 mkdir -p "${CR_DIR}"
 cp "${SHARED_DIR}/operator-credentials-request.yaml" "${CR_DIR}"
 cp "${SHARED_DIR}/controller-credentials-request.yaml" "${CR_DIR}"
-/tmp/albo/ccoctl aws create-iam-roles --name="${ACCOUNT_ROLES_PREFIX}" --region="${REGION}" --credentials-requests-dir="${CR_DIR}" --identity-provider-arn="${IDP_ARN}" --output-dir="${MANIFEST_DIR}"
+/tmp/albo/ccoctl aws create-iam-roles --name="${ACCOUNT_ROLES_PREFIX}" --region="${REGION}" --credentials-requests-dir="${CR_DIR}" --identity-provider-arn="${IDP_ARN}" --output-dir="${MANIFEST_DIR}" 2>&1 | tee "${CCOCTL_OUTPUT}"
+
+echo "=> extracting iam role arns"
+# ccoctl generates role names adding prefix, cr namespace and name,
+# this results in long names which have to be cut to fit into AWS limits.
+# that's where `aws-load-balancer-cont` pattern comes from.
+cat "${CCOCTL_OUTPUT}" | \grep -ioP 'role arn:aws:iam.* ' | \grep -v 'aws-load-balancer-cont' | cut -d' ' -f2 > ${E2E_INPUT_DIR}/${E2E_INPUT_OPERATOR_ROLE_ARN}
```

Comment:
*nit* would it be better to explicitly look for  `aws-load-balancer-operator` opposed to not `aws-load-balancer-cont`?

Reply:
Yes, I can do it. I did it this way just to make it shorter because `ccoctl` uses both the secret's namespace and name to generate the role name. As a result both roles had `aws-load-balancer-operator` substring in their names.

Reply:
Changed to the operator's role pattern.